xga/generate/esass/spec.py

```diff
@@ -451,6 +468,10 @@ def _append_spec_info(evt_list):
     rename_cmd = 'mv srctoolout_{i_no}??_{type}* {nn}'
     # Having a string to remove the 'merged' spectra that srctool outputs, even when you only request one instrument
     remove_merged_cmd = 'rm *srctoolout_0*'
+    # Commands to remove the merged files from eSASS4DR1. Will depend on instrument number. Insts 1, 2, 3, 4, 6 will use
```

Comment:
If you could add just a line of context for what the TM8 and TM9 definitions mean here, and that they aren't present in early-release eSASS, that would be handy.

Reply:
Done